mysql-test/suite/innodb/t/log_inplace_alter_with_exclusiv_lock.test

```diff
@@ -0,0 +1,31 @@
+--source include/have_debug.inc
+
+--echo #
+--echo # PS-8032 Inplace Add index with lock=exclusive doesn't generate MLOG_ADD_INDEX redo
+--echo #
+
+SET GLOBAL innodb_checkpoint_disabled=1;
+--source include/no_checkpoint_start.inc
```

Comment:
remove this checkpoint_start.inc and end.inc

Reply:
If I remove no_checkpoint_start.inc and end.inc. I get an error
exec of 'echo "restart: --debug=d,ib_log" > ' failed, error: 512, status: 2, errno: 0.
safe_process[92454]: Child process: 92455, exit: 1

Reply:
issue is something else, please check spacing etc